.github/workflows/pr_title.yml

```diff
@@ -0,0 +1,21 @@
+name: PR Title check
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+permissions:
+  pull-requests: read
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
```

Comment:
Consider adding configuration options for the semantic PR action to match the project's existing commit message standards. For example:
```yaml
- uses: amannn/action-semantic-pull-request@v5
  with:
    # Define allowed types based on your development.md guidelines
    types: |
      feat
      fix
      docs
      test
    # Add a helpful error message
    errorComment: |
      Please follow the semantic PR title format as defined in python/docs/development.md
```